styles.css

```diff
@@ -363,8 +334,270 @@ p.paragraph {
 .dev-email {
   color: #42526e;
   text-decoration: none;
-  margin-top: 0.8rem;
   font-weight: 400;
   font-size: 20px;
   line-height: 32px;
+  align-items: center;
+}
+
+.mywork-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.language-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.socials2 {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 0.8rem;
+}
+
+.desktop-nav-container {
+  visibility: hidden;
+}
+
+.background-desktop {
+  background-image: url(./tools/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 100%;
+}
+
+@media all and (min-width: 768px) {
+  .welcome {
+    width: 100%;
+    padding: 0;
+    bottom: 100px;
+    margin-top: 8rem;
+    margin-bottom: 25rem;
+  }
+
+  html,
+  body {
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+    height: 100%;
+    width: 100%;
+  }
+
+  .background-desktop {
+    background-image: url(./tools/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: initial;
+    width: 100%;
+    align-items: center;
+  }
```

Comment:
- [ ] Please kindly take note that your project is not well aligned (@min-width: 768px), kindly work  more on this as the previous reviewer pointed out.
- [ ] Moreso, the background and all other elements are not well positioned. Kindly make necessary adjustments.

## Your Design 
![127 0 0 1_5500_Mobile-portofolio_index html (1)](https://user-images.githubusercontent.com/103386183/199962053-17e77076-03f3-40b1-ad83-e72eb2a73cfd.png)

![127 0 0 1_5500_Mobile-portofolio_index html (2)](https://user-images.githubusercontent.com/103386183/199962088-bd3f4ef1-c4d0-49d2-8700-3597bf37be62.png)

Reply:
@OyePriscilla They told us to do media queries of devices that have a viewport width which is above 768px
so I don't see where it is badly aligned as I have done what they told us in the challenge